### PR TITLE
[Testing] TooltipNotes

### PR DESCRIPTION
**Pull request by Lerofni**
Initial Submission 
Plugin which lets you add custom notes to the descriptions of itemTooltips by rightclicking them. Saving notes between launches/loads is done by shipping/creating a json with those notes that gets loaded upon pluginload. Saving different notes on duplicate equipment can currently only be done by glamouring one of them(glamouring into itself is fine) 

Features to come: 
*Proper mass note editing in ImGui (currently only possible by editing the json)
*customizable note colours
*implement inventory tracking to stop needing to use the glamour workaround for duplicate equipment

### CODE REVIEW

**Comment by reiichi001** [comment]
bleatbot, approve

**Comment by Infiziert90** [comment]
Seems like something is going horrible wrong ... support is informed, so best is just to wait for now~

**Comment by philpax** [comment]
This plugin is now in the plugin review queue; we hope to get back to you soon!

**Review by Lerofni — COMMENTED** [review]
Add release changelog and update commit hash to latest(adds ability to make duplicate item id equipment unique through glamor ) Or not because there wasnt a commit

**Comment by philpax** [comment]
Just pending testing now before approval! Is it possible for you to remove the things you aren't using, like the configuration and its window?

**Comment by Lerofni** [comment]
oh yeah sure I can remove those.

**Review by philpax — APPROVED** [review]

**Comment by philpax** [comment]
Congratulations on your first plugin!
 
As a first-time contributor, if you're in the `XIVLauncher & Dalamud` Discord server and would like to get access to the private developer channels for plugin feedback and others, please let us know your Discord handle and we'll grant you the role.

**Comment by Lerofni** [comment]
Thank you! My discord handle is Lore#7178

**Pull request merged**
